assets/components/pages/project/assignments/AssignmentListItem.js

```diff
@@ -16,9 +16,9 @@ const CurationResult = ({ result }) => (
     </span>{" "}
     {result.notes && (
       <span>
-        <strong>note:</strong> {result.notes}
```

Comment:
notes and curator_comments might be long, so I suggest rendering notes and curator_comments each on separate lines to make them more legible for the curators.